tests/test_integration_tox4.py

```diff
@@ -203,6 +238,15 @@ def test_print_extras_to_file(toxenv, tmp_path):
     assert prep_tox_output(result.stdout) == expected
 
 
+@pytest.mark.parametrize("toxenv", envs_from_tox_ini())
+def test_print_dependency_groups_to_file(toxenv, tmp_path, dependency_groups_support):
+    groupspath = tmp_path / "dependency_groups"
+    result = tox("-e", toxenv, "--print-dependency-groups-to", str(groupspath))
+    assert sorted(groupspath.read_text().splitlines()) == sorted(["dg1"])
```

Comment:
Is the `sorted` on the right side necessary?
```suggestion
    assert sorted(groupspath.read_text().splitlines()) == ["dg1"]
```

Reply:
I kept the `sorted`s for symmetry, but indeed they are not necessary. I removed them both.